Examples/Python/src/RootInput.cpp

```diff
@@ -0,0 +1,91 @@
+// This file is part of the ACTS project.
+//
+// Copyright (C) 2016 CERN for the benefit of the ACTS project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "Acts/Plugins/Python/Utilities.hpp"
+#include "ActsExamples/Io/Root/RootAthenaDumpReader.hpp"
+#include "ActsExamples/Io/Root/RootAthenaNTupleReader.hpp"
+#include "ActsExamples/Io/Root/RootMaterialDecorator.hpp"
+#include "ActsExamples/Io/Root/RootMaterialTrackReader.hpp"
+#include "ActsExamples/Io/Root/RootParticleReader.hpp"
+#include "ActsExamples/Io/Root/RootSimHitReader.hpp"
+#include "ActsExamples/Io/Root/RootTrackSummaryReader.hpp"
+#include "ActsExamples/Io/Root/RootVertexReader.hpp"
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+namespace py = pybind11;
+using namespace pybind11::literals;
+
+using namespace ActsExamples;
+
+namespace Acts::Python {
+
+void addRootInput(Context& ctx) {
```

Comment:
I'm wondering if we should put these into a separate sub-package `root` now. I think @asalzburger has been thinking about this recently.

Reply:
Hmm I think is is definitively a good idea, to abstract the whole `TTree` reading/writing a bit and package it, but this goes beyond the scope of this PR I think...